user-service/README.md

```diff
@@ -27,6 +27,13 @@ host operating system account or PKI based user authentication.
 The User Service separates this concern from the rest of the system. Other components use this service's API to request user details. 
 Some deployments may also allow Palisade to add users to the system, hence the presence of the `addUser()` method in the `UserService` interface.
 
+## High Level Architecture
+
+<!--- 
+See user-service/doc/user-service.drawio for the source of this diagram
+--->
+![User Service diagram](doc/user-service.png)
```

Comment:
Can we change persistence to cache?

Reply:
The kafka bits are all better described as topics, then document usage with a verb like:
"Source from kafka topic 'request'"
"Sink to kafka topic 'user'"
"Sink to kafka topic 'error'"

Reply:
The diagram has been updated with the new descriptions